plugins/cost-insights/api-report.md

```diff
@@ -533,13 +456,6 @@ export class ExampleCostInsightsClient implements CostInsightsApi {
   getUserGroups(userId: string): Promise<Group[]>;
 }
 
-// Warning: (ae-missing-release-tag) "Group" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
-export type Group = {
```

Comment:
We are thinking of mapping these to Backstage catalog groups so it's not horrible, but could have probably been named something different to avoid confusion.